package.json

```diff
@@ -39,7 +39,7 @@
     "@bugsnag/plugin-react": "^7.17.0",
     "@floating-ui/react": "^0.19.2",
     "@opentripplanner/base-map": "^3.0.13",
-    "@opentripplanner/core-utils": "^9.0.3",
+    "@opentripplanner/core-utils": "^11.0.0",
```

Comment:
Once we're upgrading can we please also update `itinerary-body`? `yarn upgrade -S @opentripplanner --latest` should give us all we need